src/mini-cast-bar/hooks/useMiniPlayerMediaStatusListenerHook.js

```diff
@@ -0,0 +1,128 @@
+import { useEffect, useState } from "react";
+import {
+  VizbeeManager,
+  VizbeePlayerState,
+} from "../../../";
+import { ButtonType } from "../constants";
+import { LOG_TAG } from "../constants";
+
+export const useMiniPlayerMediaStatusListenerHook = ({
```

Comment:
Let's use consistent naming starting from Component folder name to hooks use MiniCastBar
say, 
1. For component folder name `VizbeeMiniCastBar`
2. For functional component name `VizbeeMiniCastBar`
3. For hooks `useMiniCastBarMediaStatusListenerHook`

Reply:
Done